src/components/events/Search.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import { TextField, List, ListItem, ListItemText } from '@material-ui/core';
+
+const Search = ({ events }) => {
```

Comment:
doesn't make sense to have this component in folder /events but call in folder /calendars